monai/apps/pathology/transforms/spatial/array.py

```diff
@@ -73,3 +75,153 @@ def get_params(self, image_size):
         )
 
         return patch_size, steps
+
+
+class TileOnGrid(Randomizable, Transform):
+    """
+    Tile the image into patches on a grid and maintain a subset of it.
+    This transform works only with np.ndarray inputs.
+
+    Args:
+        tile_count: number of tiles to extract
+        tile_size: size of the square tile
+            Defaults to ``256``.
+        tile_step: step size
+            Defaults to None (same as tile_size)
+        tile_all: Extract all non-background tiles, instead of tile_count.
+            Defaults to ``False``.
+        tile_random_offset: Randomize position of tile grid, instead of starting from the top-left corner
+            Defaults to ``False``.
+        tile_pad_full: pad image to the size evenly divisible by tile_size
+            Defaults to ``False``.
+        tile_background_val: the background constant (e.g. 255 for white background)
+            Defaults to ``255``.
+        tile_filter_mode: [min, max, None] filter tiles. If total number of tiles is more then tile_size,
```

Comment:
May change to: mode must be in `["min", "max", None]`.

Reply:
ok